src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-extension.js

```diff
@@ -161,38 +161,28 @@
              * @returns mixed
              */
             app.sandbox.sulu.getUserSetting = function(key) {
-                return (typeof app.sandbox.sulu.userSettings[key] !== 'undefined') ? app.sandbox.sulu.userSettings[key] : null;
+                return (typeof app.sandbox.sulu.userSettings[key] !== 'undefined')
+                    ? app.sandbox.sulu.userSettings[key] : null;
             };
 
             /**
              * saves data locally and to database
              * @param key
              * @param value
-             * @param url Defines where to save data to
              */
-            app.sandbox.sulu.saveUserSetting = function(key, value, url) {
+            app.sandbox.sulu.saveUserSetting = function(key, value) {
                 app.sandbox.sulu.userSettings[key] = value;
 
-                if (!url) {
-                    url = '/admin/api/users/' + SULU.user.id + '/settings/' + key;
-                }
```

Comment:
The `url`-parameter was never used, so I removed it (it also doesn't make any sense to me...)